### ISSUE TITLE
Dollar Signs ($) within comments in proto file break code generation

### ISSUE THREAD

**Issue by brandonSc** [grpc/grpc-node#1204]
### Problem description
I noticed that if I have dollars in comments in my .proto file, the code generation via `grpc_tools_node_protoc` crashes. 

### Reproduction steps
Create a proto file and add comments with dollar signs in them, e.g.:
```
// This RPC has $ dollars signs $ in the comments
rpc Test(TestRequest) returns (TestResponse)
```
run `grpc_tools_node_protoc` to create Node.js server/client code causes an error similar to the following: 
```
grpc_tools_node_protoc --js_out=import_style=commonjs,binary:./ --grpc_out=store-grpc-clients/nodejs --plugin=protoc-gen-grpc=`which grpc_tools_node_protoc_plugin` api/grpc/store/store.proto
[libprotobuf FATAL /tmpfs/src/github/grpc-node/packages/grpc-tools/deps/protobuf/src/google/protobuf/io/printer.cc:153]  Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():   Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 
/usr/local/lib/node_modules/grpc-tools/bin/protoc_plugin.js:37
    throw error;
    ^

Error: Command failed: /usr/local/lib/node_modules/grpc-tools/bin/grpc_node_plugin
[libprotobuf FATAL /tmpfs/src/github/grpc-node/packages/grpc-tools/deps/protobuf/src/google/protobuf/io/printer.cc:153]  Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():   Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 

    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
--grpc_out: protoc-gen-grpc: Plugin failed with status code 1.
/usr/local/lib/node_modules/grpc-tools/bin/protoc.js:41
    throw error;
    ^

Error: Command failed: /usr/local/lib/node_modules/grpc-tools/bin/protoc --plugin=protoc-gen-grpc=/usr/local/lib/node_modules/grpc-tools/bin/grpc_node_plugin --js_out=import_style=commonjs,binary:./ --grpc_out=store-grpc-clients/nodejs --plugin=protoc-gen-grpc=/usr/local/bin/grpc_tools_node_protoc_plugin api/grpc/store/store.proto
[libprotobuf FATAL /tmpfs/src/github/grpc-node/packages/grpc-tools/deps/protobuf/src/google/protobuf/io/printer.cc:153]  Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():   Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 
/usr/local/lib/node_modules/grpc-tools/bin/protoc_plugin.js:37
    throw error;
    ^

Error: Command failed: /usr/local/lib/node_modules/grpc-tools/bin/grpc_node_plugin
[libprotobuf FATAL /tmpfs/src/github/grpc-node/packages/grpc-tools/deps/protobuf/src/google/protobuf/io/printer.cc:153]  Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 
terminate called after throwing an instance of 'google::protobuf::FatalException'
  what():   Undefined variable: 1.99), (30 Day, 60 min talk or 180 sms, 

    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
--grpc_out: protoc-gen-grpc: Plugin failed with status code 1.

    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
```

### Environment
 - Ubuntu 18.04.3 LTS amd64
 - Node v8.10.0
- `grpc_tools_node_protoc --version` prints: libprotoc 3.6.1

**Comment by murgatroid99** [comment]
I think I know what's happening here. In the [generator code](https://github.com/grpc/grpc-node/blob/master/packages/grpc-tools/src/node_generator.cc) the dollar sign is used for variable interpolation, and the comments aren't interpolated.

**Comment by murgatroid99** [comment]
The fix for this has been published in grpc-tools version 1.8.1.
